README.md

```diff
@@ -112,16 +81,19 @@ The following options are specific to appmetrics:
 
 ## Running Node Application Metrics
 
-### The `node-hc` command
-If you [globally installed](#installation) this module with npm, you can use the `node-hc` command to run your application instead of the `node` command. This will run your application as it would normally under node (including any node options) but additionally load and start `appmetrics`.
+### Preloading appmetrics
+In previous versions appmetrics came with an executable, `node-hc`, which could be used instead of the `node` command to run your application and load and start appmetrics. This has been removed in version 4.0.0, instead you can use:
 
 ```sh
-$ node-hc app.js
+$ node --require appmetrics/start app.js
 ```
+to preload and start appmetrics, or in Node.js 8.0.0 or later, use the NODE_OPTIONS environment variable:
```

Comment:
It's also in `6.12.0` (and onwards), so maybe

`Node.js 8.0.0, 6.12.0, or later`

or something

Reply:
done